### ISSUE TITLE
Index has links that are going nowhere

### ISSUE THREAD

**Issue by eddelbuettel** [rstudio/r-manuals#3]
Thanks for putting this together as a `quarto` demo -- it's quite compelling.

I just tried to help somebody by looking up / referencing `R_LIBS_USER` which does of course exist in the index for the R Inst + Admin manual -- but its link

     https://rstudio.github.io/r-manuals/r-admin/Add_002don-packages.html#index-r_005flibs_005fuser

gets us nowhere.  It should probably go to 

     https://rstudio.github.io/r-manuals/r-admin/Add-on-packages.html#managing-libraries

Would be sweet if you can massage that into working...

**Comment by cderv** [comment]
Thanks for reporting. We'll fix this !

**Comment by cderv** [comment]
I have fixed part of the issue. The index pages are tables with two links, one for the term directly and another for the section  in which the term is. I fixed the latter. The second link will correctly send to the right section (so `Managing libraries` in the example given.)
The former still need to be supported - we currently do not keep correctly the index term ids in our process. So it will open the HTML page but not in the place the term is. Still something to be fixed part of #1
